spec/i.appendix-a.md

```diff
@@ -33,6 +33,49 @@ Then, we compose React containers to fetch data from different sources and load
+
+* Full ES2015 (ES6) modules support. Ben Newman has written a good introduction to
+[Using JavaScript modules in Meteor](https://github.com/meteor/meteor/blob/release-1.3/packages/modules/README.md)
+* Better [npm](https://www.npmjs.com) (Node Package Manager) integration.
+* Improved control over file load order.
+
+### Mantra and Meteor
+
+Mantra uses some of the these technologies a bit differently. For an example, Meteor's React tutorial suggests using a mixin to access Mongo collection data. But Mantra uses a container, which is the modern way to use React.
+
+Keep this in mind as you read, and as you notice other examples help out with an issue or a pull request.
+
+### Key Meteor Concepts
+
+If you are new to Meteor, focus on understanding these key concepts which differentiate Meteor from some of the tools that you may be more familiar with.
+
+#### Shared client & server code
+
+With the exception of code specifically targeted at the server or client Meteor applications run the same code on the client and server sides of the app. This means that, in general, you write code once and don't need to worry about making the bridge between client and server – Meteor handles that for you.
+
+But, since the client and server contexts are not identical, you need to keep potential differences in mind. For example:
+
+* Code that needs to run in a browser context needs to run only on the client – it makes no sense to show an alert or go to a screen position on the server.
+* Code whose context depends on the current user also needs to run on the client – the server is responding to, potentially, tens or maybe even thousands of different users and the notion of the "current" user doesn't make sense there.
+* Secure code, on the other hand, must run on the server.
+
+#### Reactivity
+
+Meteor apps are **reactive** – changes to the state of the app are immediately visible to all users, there is no need to constantly refresh the page in your browser to see the current state.
+
+#### Collections
+
+Meteor uses **collections** to store persistent data. Nothing special needs to be done to keep the client and server view of the collection coherent – collections update automatically, so each client (and the server) always sees an up-to-date view of the data.
+
+#### Reading List
+
+* [Meteor for Front-End Engineers](https://davidwalsh.name/meteor-frontend-engineers)
+
+* [Learn Meteor Fundamentals and Best Practices](http://andrewscala.com/meteor/) - this article from, 2012, is dated. However it has a good explanation of the traditional Meteor directory structure. Mantra is opinionated about how to structure a Meteor app, it is worth understanding the differences here. Pay attention to the
```

Comment:
I wish this is better than this one: http://meteortips.com/first-meteor-tutorial/